build/gulpfile.hygiene.js

```diff
@@ -391,6 +413,8 @@ function hygiene(some) {
 	const productJsonFilter = filter('product.json', { restore: true });
 
 	const result = input
+		.pipe(filter(fileLengthFilter))
```

Comment:
I'd suggest splitting this out into a separate step like we do for the formatting (although there's no need to store the returned stream since we wouldn't chain anything off of it) https://github.com/microsoft/azuredatastudio/blob/4a68ab46590134be0fa663cb59ac59b8603df1d1/build/gulpfile.hygiene.js#L430

The issue with this is that now anything added to the file length filter is going to also be filtered out of doing all the other checks in this chain - which isn't necessarily what we want.

Reply:
It's either I let all the rule filters applied to the length rule or other way around,, I don't see a way around it. So what's the right way to go here?

Reply:
Are you still blocked on this after our discussion? Just piping the input through  the filter and function alone should be enough.

Reply:
yea, that didn't work :(